src/skeleplex/graph/skeleton_graph.py

```diff
@@ -249,8 +222,19 @@ class SkeletonGraph:
 
     _backend = "networkx"
 
-    def __init__(self, graph: nx.Graph):
+    def __init__(
+        self,
+        graph: nx.Graph,
+        origin: int | None = None,
+        image_path: str | None = None,
+        image_key: str | None = None,
+        voxel_size: float | None = None,
+    ):
         self.graph = graph
+        self.origin = origin
+        self.image_path = image_path
+        self.image_key = image_key
+        self.voxel_size = voxel_size
```

Comment:
I think we should call this `voxel_size_um` to make it clear that the unit should be microns. Alternatively, we can explain it in the docstring, but as a developer, I think it is nice when it is in the variable name.

If you change it, please change the` __init__` argument name as well.

I'm not sure if you have seen it, but VS Code and PyCharm both have a function to rename variables that will rename references throughout the code base - much easier than changing them all by hand!

```suggestion
        self.voxel_size_um = voxel_size
```